### ISSUE TITLE
Flexbox

### ISSUE THREAD

**Issue by NoreenNaz1234** [Farheen-Shadani/Test-9-May#2]
good to see you used flex box and some how it is responsive,, but need some changes. ap mock up ko dkhen aur change kren. 

https://github.com/Farheen-Shadani/Test-9-May/blob/df54817bc1251db2a1e5ae1ff999008f02fa74ec/index.html#L41

**Comment by Farheen-Shadani** [comment]
yes miss net issues the to mein itna detail mein flex box ni par pai thi... to jitna thora bht ata tha mene use kia ab in sha Allah mein isko dubara se jo zara sa issue he wo b resolve kardeti hun... thanks...
